packages/zend-db/library/Zend/Db/Adapter/Mysqli/Exception.php

```diff
@@ -37,4 +37,16 @@
  */
 class Zend_Db_Adapter_Mysqli_Exception extends Zend_Db_Adapter_Exception
 {
+    public static function fromMysqliException($exception)
```

Comment:
typing could be added: `public static function fromMysqliException(mysqli_sql_exception $exception)`